Blockstack/Classes/Blockstack.swift

```diff
@@ -9,23 +9,27 @@ import Foundation
 
 open class Blockstack {
     
-    private var instance: BlockstackInstance?
+    private var authenticationModule: BlockstackAuthentication
+    private var storageModule: BlockstackStorage
     
-    private static var sharedBlockstack: Blockstack = {
+    open static var shared: Blockstack = {
         let blockstack = Blockstack()
         return blockstack
     }()
```

Comment:
User can use the `shared` instance. I really liked that functionality in "Realm".

Renamed it to `shared` so the user calls: `Blockstack.shared` instead of `Blockstack.sharedBlockstack` - removes redundancy.